tldextract/tldextract.py

```diff
@@ -229,7 +226,7 @@ def _get_tld_extractor(self):
         if self._extractor:
             return self._extractor
 
-        tlds = frozenset()
```

Comment:
Oh, I wasn't suggesting to ditch `frozenset`. We [still need its performance](https://github.com/john-kurkowski/tldextract/blob/038217b79f378da8b055e30d65a1b83e29926514/tldextract/tldextract.py#L352). I meant, coerce from `frozenset` to something JSON serializable, like `list`, _when_ you need to serialize.